action.yml

```diff
@@ -11,6 +11,15 @@ inputs:
   token:
     required: true
     description: 'GitHub token from secrets.GITHUB_TOKEN'
+  allow-external-github-orgs:
+    required: false
+    description: 'Allow installing from external GitHub organizations'
```

Comment:
I think `action.yml` is used to give information about how the action is used, so it would warn that an unknown argument was used when `allow-external-github-org` was set. AFAIK, it doesn't affect any functionality